src/java.base/share/classes/java/util/concurrent/locks/AbstractQueuedLongSynchronizer.java

```diff
@@ -271,14 +293,17 @@ final int acquire(Node node, long arg, boolean shared,
         Thread current = Thread.currentThread();
         byte spins = 0, postSpins = 0;   // retries upon unpark of first thread
         boolean interrupted = false, first = false;
-        Node pred = null;                // predecessor of node when enqueued
+        Node pred = null, t;             // predecessor of node when enqueued
```

Comment:
Nit: please don't use this style of multi-variable declaration as they are very easy to mis-read. Also the comment only applies to one of the variables.

Reply:
OK, changed.